incubator/bootstrap_cli/__main__.py

```diff
@@ -240,20 +253,20 @@ def acl_template(actions, scope):
         return {"actions": actions, "scope": scope}
 
     @staticmethod
-    def get_allprojects_name_template(group_type=None):
-        return f"{group_type}:allprojects" if group_type else "allprojects"
+    def get_allprojects_name_template(group_ns=None):
+        return f"{group_ns}:{BootstrapCore.AGGREGATED_GROUP_NAME}" if group_ns else BootstrapCore.AGGREGATED_GROUP_NAME
 
     @staticmethod
     def get_dataset_name_template():
         # 211013 pa: remove env prefixes
-        # return f"{shared_global_config['env']}:" + "{group_prefix}:dataset"
-        return "{group_prefix}:dataset"
+        # return f"{shared_global_config['env']}:" + "{group_core}:dataset"
```

Comment:
is this still needed, or can it be removed?

Reply:
the comment? I've tried to cleanup the old `shared_global_config` references, but missed that

Reply:
Is it ok If I remove the rest and update the PR before merging?